src/parsers/in.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import P from 'parsimmon'
+import crap from './crap'
+
+import type { TupleNodeType, PrimitiveNodeType } from '../types'
+
+import StringParser from './primitives/string'
+import ObjectParser from './object'
+
+export default P.seq(
+  StringParser.trim(crap),
+  P.regexp(/\s*in\s*/).trim(crap),
+  ObjectParser.trim(crap).map(
+    // $FlowFixMe
+    (found: mixed): [TupleNodeType] => found.value[0].value
+  )
+)
+  .map(([key, _, props]: [PrimitiveNodeType, mixed, [TupleNodeType]]): boolean => {
```

Comment:
It's not required to execute code in the parser. Instead we generate an AST. (See `NodeType` for it's format).

We have separate generators that convert AST into JavaScript, and they reside in `./src/generators/`.